src/component/tool/FloatMoleculeStructures/index.tsx

```diff
@@ -47,6 +47,7 @@ export function FloatMoleculeStructuresInner(
               ranges,
               activeTab,
               displayerMode,
+              position: molecule.position,
               molecule: molecules.find(
```

Comment:
Not in the scope of this PR but just to mention one of our "best practices":

Instead of using `as`, create a variable and `assert` it is defined since it should be an error for the molecule not to exist.

I don't know if nmrium already has some typescript-friendly assert utility functions but I can provide them if needed.

If one day the error happens, it will be easy to know exactly where the error stems from from the stack trace

Reply:
Thank you
it's fixed now